### PR TITLE
Use tox to run tests and build local docs

### PR DESCRIPTION
**Pull request by vasilty**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,3 +1,24 @@
+[tox]
+envlist = py36,py37,py38,flake8
+
+[testenv]
+deps = -rrequirements.txt
```
Comment (i-trofimtschuk) [inline tox.ini:5]: @vasilty i think this is missing a space

Comment (vasilty) [reply tox.ini:5]: no, it's ok so. it is still understood and doesn't pose problems

Comment (i-trofimtschuk) [reply tox.ini:5]: i still find it confusing and would add the space if it also works with a space

Comment (i-trofimtschuk) [reply tox.ini:5]: apart from that PR LGTM

**Comment by i-trofimtschuk** [comment]
@vasilty FYI conflict with master in requirements.txt ;)

File: tox.ini [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+commands =
+    pytest {posargs:--cov redis_tasks --run-slow}
+
+[testenv:flake8]
+basepython = python
+skip_install = true
+commands =
+    flake8 .
+
+[testenv:docs]
```
Comment (i-trofimtschuk) [inline tox.ini:15]: @vasilty do we need to execute this automatically somewhere?
or is it enough that we have readthedocs? i thought about travis

**Comment by i-trofimtschuk** [comment]
@vasilty can you check my last merge commit?

File: tox.ini (hunk 2 above)
Comment (vasilty) [reply tox.ini:15]: This builds the docs, do we need to build them on CI? I think it is fne to do this manually when needed

File: tox.ini (hunk 1 above)
Comment (vasilty) [reply tox.ini:5]: @i-trofimtschuk https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints this is from the docs

**Comment by vasilty** [comment]
LGTM

**Pull request merged**